game/src/contrib/systems/HealthbarSystem.java

```diff
@@ -0,0 +1,92 @@
+package contrib.systems;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.math.Vector3;
+import com.badlogic.gdx.scenes.scene2d.ui.Container;
+import com.badlogic.gdx.scenes.scene2d.ui.ProgressBar;
+
+import contrib.components.HealthComponent;
+
+import core.Entity;
+import core.System;
+import core.components.PositionComponent;
+import core.components.UIComponent;
+import core.systems.CameraSystem;
+import core.utils.Point;
+import core.utils.logging.CustomLogLevel;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Logger;
+
+public final class HealthbarSystem extends System {
+    private static final Logger LOGGER = Logger.getLogger(HealthbarSystem.class.getSimpleName());
+
+    private final Map<Integer, ProgressBar> healthbarMapping = new HashMap<>();
+
+    public HealthbarSystem() {
+        super(HealthComponent.class, PositionComponent.class);
+        this.onEntityAdd =
+                (x) -> {
+                    LOGGER.log(CustomLogLevel.TRACE, "HealthbarSystem got send a new Entity");
+                    ProgressBar newHealthbar =
+                            createNewHealthbar(x.fetch(PositionComponent.class).orElseThrow());
+                    LOGGER.log(CustomLogLevel.TRACE, "created a new Healthbar");
+                    Entity e = new Entity();
```

Comment:
```suggestion
                    Entity e = new Entity("Healthbar");
```

du kannst Entitäten einen Namen geben, macht beim Logging etwas einfacher.